templates/user/my_account.html.twig

```diff
@@ -8,7 +8,7 @@
         <span class="settings__username"><strong>Email</strong> {{ app.user.emailaddress }}</span><br />
         <span class="settings__username"><strong>Code postal</strong> {{ app.user.postAddress.postalCode }}</span><br/>
 
-        {% if app.user.activatedAt %}
+        {% if has_role_adherent and app.user.activatedAt %}
```

Comment:
just wondering if `{% if has_role_adherent %}` wouldn't be enough here?
can a `User` be `Adherent` without prior activation ?

Reply:
There are a test where he check if an adherent without activation can see the page as a non adherent. It s weird :s